.github/workflows/deploy.yml

```diff
@@ -12,14 +16,13 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Setup Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
 
       - name: Install Bootstrap
         run: npm install
 
-      - uses: jeffreytse/jekyll-deploy-action@v0.5.0
+      - uses: jeffreytse/jekyll-deploy-action@v0.5.1
```

Comment:
Since we don't know when will the fix being merged in the upstream, we can use `ambersun1234/jekyll-deploy-action@52f40044bb2a2b5d7365472ed640bfd412689d41` as a temporary solution